python/lsst/ap/association/loadDiaCatalogs.py

```diff
@@ -42,12 +42,6 @@ class LoadDiaCatalogsConfig(pexConfig.Config):
         default=250,
         min=0,
     )
-    loadDiaSourcesByPixelId = pexConfig.Field(
-        doc="Load DiaSources by their HTM pixelId instead of by their "
-            "associated diaObjectId",
-        dtype=bool,
-        default=True,
-    )
```

Comment:
I should have probably squashed all commits 🙂 as it's not easy to test them separately now. I'll move this patch to an earlier commit.

Reply:
Looking at all changes I think this is the right commit for deleting it. When I switched to regions instead of pixel ranges there were still two methods (getDiaSourcesInRegion and getDiaSources). I should have probably renamed the option to `loadDiaSourcesByRegion` but I knew it was going to disappear anyways so I kept it.

I still can squash everything if you prefer not to see this messy history. The total diff is not that big.

Reply:
So, OK to squash?

Reply:
It's up to you. Personally I found the commit division refreshingly clear overall, so I wouldn't mind keeping it.

Reply:
OK, no squashing then.